smartsim/_core/launcher/step/mpiStep.py

```diff
@@ -46,8 +45,20 @@ def __init__(self, name, cwd, run_settings):
         :param run_settings: run settings for entity
         :type run_settings: RunSettings
         """
+
         super().__init__(name, cwd)
+
+        self._supported_launchers = [
+            "PBS",
+            "COBALT",
+            "SLURM",
+            "LSB"
+        ]
+
+
         self.run_settings = run_settings
+        self._run_command = self.run_settings._run_command
```

Comment:
Thoughts on making this a property?
```py
@property
def run_command(self):  # or _run_command if we prefer it be a "protected" attr
   return self.run_settings._run_command
```
That way its still "un-writable" like the like the protected attr, but that way if the run command were to be changed, we wouldn't have to update it in two locations (`_BaseMPIStep._run_command` and `_BaseMPIStep.run_settings._run_command`)

Slightly on the fence about this suggestion bc I cannot think of why the run command of a step would ever change, but I figured its better to protect it early

Reply:
I think that should be ok, I'm going to meditate on this overnight, but will probably take your suggestion

Reply:
Yep, after my morning coffee; I'm more onboard. Might be a little overly defensive, but better than someone accidentally stumbling their way to this.